### PR TITLE
Add a field that displays the event creator and make company name a link

### PR DESCRIPTION
**Pull request by SmithPeder**
Depends on https://github.com/webkom/lego/pull/1019

### CODE REVIEW

File: app/routes/events/components/EventDetail/index.js [hunk 1]
```diff
@@ -141,7 +142,19 @@ export default class EventDetail extends Component<Props> {
...
         key: 'Arrangerende bedrift',
-        value: event.company.name
+        value: (
+          <Link to={`/companies/${event.company.id}`}>
+            {event.company.name}
+          </Link>
+        )
+      },
+      event.createdBy && {
+        key: 'Ansvarlig',
```
Comment (finnss) [inline app/routes/events/components/EventDetail/index.js]: Is the creator of an event always responsible for it? Does this also apply to events for interest groups and other such non-standard cases? Not a huge deal of course, but I can imagine Bedkom requesting the ability to change who's responsible if we include this functionality

Comment (SmithPeder) [reply app/routes/events/components/EventDetail/index.js]: @edvardvb Finn got a point here

Comment (edvardvb) [reply app/routes/events/components/EventDetail/index.js]: Could just use "forfatter", but I believe we used a similar term on nerd, and Bedkom never had an issue with that. Maybe check the old repo and use the same term?

Comment (SmithPeder) [reply app/routes/events/components/EventDetail/index.js]: Digging through NERD was to hard. So i agree that we can just go with "forfatter", and if Bedkom wants something else we'll just change it

**Pull request merged**